src/ex4/client/src/screens/home/Home.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'
+import Card from '../../components/card/Card'
+import AddTask from '../../components/add_task/AddTask'
+import TaskList from '../../components/task_list/TaskList'
+import styles from './Home.module.css'
+import Loader from '../../components/loader/Loader'
+
+function Home() {
+
+    const [itemsFlag, setItemsFlag] = useState(true)
+    const [loading, setLoading] = useState(false)
+
+    return (
+        <div className={styles.home_container}>
+            {loading && <Loader />}
```

Comment:
adding a loader is really nice! think about the user experience though, when your loader is shown it's over the whole screen and the user can't do anything.. maybe if the loader was smaller the user could still do some other actions.